src/judoscale/core/reporter.py

```diff
@@ -49,7 +61,8 @@ def get_metrics(self):
 
     def _report_metrics(self):
         metrics = metrics_store.flush()
-        api_client.post_report(self._build_report(metrics))
+        if metrics:
```

Comment:
We want to post a report even if `metrics` is an empty list.

This lets the Judoscale API know that the client (reporter) is active, just not receiving requests.